dev/dependencyList

```diff
@@ -72,6 +72,10 @@ jetty-util/9.4.41.v20210516//jetty-util-9.4.41.v20210516.jar
 jline/0.9.94//jline-0.9.94.jar
 libfb303/0.9.3//libfb303-0.9.3.jar
 libthrift/0.9.3//libthrift-0.9.3.jar
+log4j-1.2-api/2.17.1//log4j-1.2-api-2.17.1.jar
+log4j-api/2.17.1//log4j-api-2.17.1.jar
+log4j-core/2.17.1//log4j-core-2.17.1.jar
+log4j-slf4j-impl/2.17.1//log4j-slf4j-impl-2.17.1.jar
 log4j/1.2.17//log4j-1.2.17.jar
```

Comment:
We should remove `log4j-1.2` from the runtime jar

Reply:
sorry, what it is the runtime jar?

Reply:
I mean the jars under the `$KYUUBI_HOME/jars`

Reply:
I think `log4j-1.2.17.jar` should not existed after your change, but why it still here?

Reply:
I checked that `log4j-1.2.17.jar` is not existing in `$KYUUBI_HOME/jars`.

Reply:
```
[INFO] org.apache.kyuubi:kyuubi-assembly_2.12:pom:1.5.0-SNAPSHOT
[INFO] +- org.apache.kyuubi:kyuubi-common_2.12:jar:1.5.0-SNAPSHOT:compile
[INFO] |  +- org.scala-lang:scala-library:jar:2.12.15:compile
[INFO] |  +- org.slf4j:slf4j-api:jar:1.7.30:compile
[INFO] |  +- org.slf4j:jcl-over-slf4j:jar:1.7.30:compile
[INFO] |  +- org.slf4j:slf4j-log4j12:jar:1.7.30:compile
[INFO] |  |  \- log4j:log4j:jar:1.2.17:compile
[INFO] |  +- commons-codec:commons-codec:jar:1.15:compile

```

I do not know why the org.slf4j:slf4j-log4j12:jar:1.7.30 is transferred.